patches/tModLoader/Terraria.Social/SocialAPI.cs.patch

```diff
@@ -1,13 +1,19 @@
 --- src/Terraria\Terraria.Social\SocialAPI.cs
 +++ src/tModLoader\Terraria.Social\SocialAPI.cs
-@@ -29,6 +_,9 @@
+@@ -28,14 +_,11 @@
+ 			if (!mode.HasValue)
  			{
  				mode = new SocialMode?(SocialMode.None);
- #if CLIENT
-+#if DEBUG
-+                if (!Program.LaunchParameters.ContainsKey("-nosteam"))
-+#endif
- 				mode = new SocialMode?(SocialMode.Steam);
- #else
+-#if CLIENT
+-				mode = new SocialMode?(SocialMode.Steam);
+-#else
++
  				if (Program.LaunchParameters.ContainsKey("-steam"))
+ 				{
+ 					mode = new SocialMode?(SocialMode.Steam);
+ 				}
+-#endif
```

Comment:
This roll back will not allow GoG exe to run until we introduce md5 check. Is that the intention?
Otherwise I can fix the DEBUG section (that I probably broke) and add few lines to still be able to run GoG build with "-nosteam" key.